### DISCUSSION TITLE
Flux

### DISCUSSION THREAD

**Discussion by liksar34** [lllyasviel/Fooocus#3776] [Ideas]
Good afternoon. Will there be support for flux models? Very interesting how these models would work in focus, besides flux is very popular.

**Comment by IPv6** [comment]
No. popularity alone is not enough to add something complicated

**Reply by poor7** [reply]
If Flux has already been added to 2 popular forks, then it is not such a difficult task.

**Reply by wisdombox44** [reply]
ComfyUI or SwarmUI,

**Reply by IPv6** [reply]
> If Flux has already been added to 2 popular forks, then it is not such a difficult task.

Current implementations are based on diffusers, so all heavy lifting implemented elsewhere.
Foooocus is not based on diffusers, so this option is not available. This is partial solution without extensive integration

SimlpeSDXL fork is a fooocus with Flux support based on diffusers, so you can try it
